jenkins/e2e-image/e2e-runner.sh

```diff
@@ -176,6 +176,11 @@ function install_google_cloud_sdk_tarball() {
     mkdir -p "${install_dir}"
     tar xzf "${tarball}" -C "${install_dir}"
 
+    if running_in_docker; then
+        # Delete any previously installed SDK to prevent weirdly overlapping
+        # installs from fighting each other.
+        rm -rf "${install_dir}/google-cloud-sdk"
+    fi
```

Comment:
emmm sounds easier just get rid of gcloud from kubekins-e2e image, since you need to bump the image anyway